assets/js/modules/tagmanager/components/setup/SetupFormInstructions.js

```diff
@@ -0,0 +1,86 @@
+import ErrorText from '../../../../components/error-text';
+import { FormInstructions } from '../common';
+const { useSelect } = Data;
+
+export default function SetupFormInstructions() {
+	const singleAnalyticsPropertyID = useSelect( ( select ) => select( STORE_NAME ).getSingleAnalyticsPropertyID() );
+	const hasMultipleAnalyticsPropertyIDs = useSelect( ( select ) => select( STORE_NAME ).hasMultipleAnalyticsPropertyIDs() );
+	const analyticsPropertyID = useSelect( ( select ) => select( MODULES_ANALYTICS ).getPropertyID() );
+	const analyticsModuleActive = useSelect( ( select ) => select( CORE_MODULES ).isModuleActive( 'analytics' ) );
+
+	if ( [ singleAnalyticsPropertyID, hasMultipleAnalyticsPropertyIDs, analyticsPropertyID, analyticsModuleActive ].includes( undefined ) ) {
+		return <FormInstructions />;
+	}
+
+	// Multiple property IDs implies secondary AMP where selected containers don't reference the same Analytics property ID.
+	if ( hasMultipleAnalyticsPropertyIDs ) {
+		const message = __( 'Looks like you’re already using Google Analytics within your Google Tag Manager configurations. However, the configured Analytics tags reference different property IDs. You need to configure the same Analytics property in both containers.', 'google-site-kit' );
+
+		return <ErrorText message={ message } />;
+	}
+
+	if ( analyticsModuleActive ) {
+		// If the Analytics module is active, and selected containers reference a different property ID
+		// than is currently set in the Analytics module, display an error explaining why the user is blocked.
+		if ( singleAnalyticsPropertyID && singleAnalyticsPropertyID !== analyticsPropertyID ) {
+			/* translators: %1$s: Tag Manager Analytics property ID, %2$s: Analytics property ID */
+			const message = __( 'Looks like you’re already using Google Analytics within your Google Tag Manager configuration. However, its Analytics property %1$s is different from the Analytics property %2$s, which is currently selected in the plugin. You need to configure the same Analytics property in both places.', 'google-site-kit' );
+
+			return <ErrorText message={ sprintf( message, singleAnalyticsPropertyID, analyticsPropertyID ) } />;
+		}
+		// If the Analytics module is active, and the Analytics property ID in GTM
+		// matches the property ID configured for the Analytics module,
+		// inform the user that GTM will take over outputting the tag/snippet.
+		if ( analyticsModuleActive && singleAnalyticsPropertyID && singleAnalyticsPropertyID === analyticsPropertyID ) {
+			/* translators: %s: Analytics property ID */
+			const message = __( 'Looks like you’re using Google Analytics. Your Analytics property %s is already set up in your Google Tag Manager configuration, so Site Kit will switch to using Google Tag Manager for Analytics.', 'google-site-kit' );
+			return (
+				<p>
+					{ sprintf( message, singleAnalyticsPropertyID ) }
+				</p>
+			);
+		}
```

Comment:
With the above, this can essentially become the `else`, so the wrapping clause here can be removed.